ElasticsearchReportHelper.php

```diff
@@ -657,7 +658,32 @@ public static function permissionFilters(array $options) {
       'class' => 'permissions-filter',
     ];
 
-    return data_entry_helper::select($controlOptions);
+    $dropdown = data_entry_helper::select($controlOptions);
+    $html = <<<HTML
+<div>
+  $dropdown 
+  <div id="permission-filters-notice"></div>
+</div>
+
+HTML;
+
+  helper_base::$indiciaData['esPermissionFilterNotices'] = json_encode($options['notices']);
```

Comment:
If you look at getControlContainer, you'll see that we've standardised on using a data attribute data-idc-config on the control container to pass options in, rather than attaching them to the global indiciaData.

Reply:
Removed inline JS to plugin and used data attribute to store notices.